lms/static/js/learner_dashboard/views/explore_new_programs_view.js

```diff
@@ -0,0 +1,44 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
```

Comment:
maybe a dumb question, but gettext doesn't ever seem to be used in this scope; is it unnecessary?